nospoiler/nospoiler.py

```diff
@@ -323,40 +378,47 @@ async def warn(self, ctx: commands.Context) -> None:
             await ctx.send("Spoiler warning message is now enabled.")
 
     @nospoiler.command()
-    async def warnmessage(self, ctx: commands.Context, *, message: str) -> None:
-        """Set the spoiler warning message."""
-        if len(message) > 1024 or len(message) < 1:
-            return await ctx.send("Message must be between 1 and 1024 characters.")
-        await self.config.guild(ctx.guild).spoiler_warn_message.set(message)
-        await ctx.send("Spoiler warning message has been set.")
-
-    @nospoiler.command(aliases=["resetwarnmsg", "resetwarn"])
-    async def resetwarnmessage(self, ctx: commands.Context) -> None:
-        """Reset the spoiler warning message to default."""
-        await self.config.guild(ctx.guild).spoiler_warn_message.set(
-            "Usage of spoiler is not allowed in this server."
-        )
-        await ctx.send("Spoiler warning message has been reset.")
+    async def warnmessage(
+        self, ctx: commands.Context, *, message: TagscriptConverter
+    ) -> None:
+        """
+        Set the spoiler warning message.
 
-    @nospoiler.command()
-    async def embed(self, ctx: commands.Context) -> None:
-        """Toggle the spoiler warning message embed."""
-        if not ctx.bot_permissions.embed_links:
-            self.log.info(
-                f"I cannot toggle the spoiler warning message embed because I don't have embed_links permission in {ctx.channel.mention}."
-            )
-            return await ctx.send(
-                "I don't have embed_links permission to let you toggle the spoiler warning message embed."
+        (Supports Tagscript)
+
+        **Blocks:**
+        - [Assugnment Block](https://phen-cogs.readthedocs.io/en/latest/tags/tse_blocks.html#assignment-block)
+        - [If Block](https://phen-cogs.readthedocs.io/en/latest/tags/tse_blocks.html#if-block)
+        - [Embed Block](https://phen-cogs.readthedocs.io/en/latest/tags/parsing_blocks.html#embed-block)
+        - [Command Block](https://phen-cogs.readthedocs.io/en/latest/tags/parsing_blocks.html#command-block)
+
+        **Variable:**
+        - `{server}`: [Your guild/server.](https://phen-cogs.readthedocs.io/en/latest/tags/default_variables.html#server-block)
+        - `{member}`: [Author of the message.](https://phen-cogs.readthedocs.io/en/latest/tags/default_variables.html#author-block)
+        - `{color}`: [botname]'s default color.
+
+        **Example:**
+        ```
+        {embed(description):{member(mention)} You cannot send spoilers here.}
+        {embed(color):{color}}
+        ```
```

Comment:
I think its fair to show title too.
```suggestion
        ```
        {embed(title):No spoiler allowed.}
        {embed(description):{member(mention)} You cannot send spoilers here.}
        {embed(color):{color}}
        ```
```